chart/README.md

```diff
@@ -541,14 +541,63 @@ Some example commands are shown below.
    sonatype/nexus-iq-server-ha --version <version>
    ```
 
+### Autoscaling
+
+IQ HA helm chart includes support for Kubernetes Horizontal Pod Autoscaling (HPA). With this enabled you can set the
+cluster to automatically scale up/down based on the resource demand such as CPU and Memory utilization.
+
+Note: When setting auto-scaling parameters please make sure to have sufficient hardware resources available in the
+underlying nodes meet the max pod demands.
+HPA is disabled by default. If you want to enable it, you need to set the `hpa.enabled` parameter to `true`.
```

Comment:
missing how to override utilization values and how to do so

Reply:
What do you mean? The resource definition parameters are explained here: https://github.com/sonatype/nexus-iq-server-ha/blob/4412336e41844b2428b0ea0be86f6958fda0d815/chart/README.md?plain=1#L641C1-L644C44